link.go

```diff
@@ -73,8 +77,11 @@ func (link *ToxicLink) Start(name string, source io.Reader, dest io.WriteCloser)
 				"err":   err,
 			}).Warn("Source terminated")
 		}
+		if server.Metrics.proxyMetricsEnabled() {
+			server.Metrics.ProxyMetrics.ReceivedBytesTotal.WithLabelValues(linkLabels...).Add(float64(bytes))
+		}
 		link.input.Close()
-	}()
+	}(labels)
```

Comment:
I see usage of `server.Metrics` inside goroutine. You can specify either `server` or `server.Metrics` in arguments.